### ISSUE TITLE
hyperparameters tuning

### ISSUE THREAD

**Issue by Erhushenshou** [ubc-vision/3dgs-mcmc#35]
Hi, thanks for your wonderful innovation. I'm curious to know how we coulde tune the hyper-parameters like the learning rate and the noise term in mcmc framework. Thank you.
